samples/server/go-server/api/server/storage/pgsqlstore.go

```diff
@@ -472,3 +490,23 @@ func (pg *pgSQLStore) ListOperations(pID, filters string) ([]*opspb.Operation, e
 	}
 	return ops, nil
 }
+
+// Encrypt int64 using provided key
+func encryptInt64(v int64, key string) (string, error) {
+	k, _ := fernet.DecodeKey(key)
+	bytes, err := fernet.EncryptAndSign([]byte(strconv.FormatInt(v, 10)), k)
+	if err != nil {
+		return "", err
+	}
+	return string(bytes), nil
```

Comment:
we should encode this in base64

Reply:
The token returned from fernet-go is already base64 encoded.